### ISSUE TITLE
ios safari bug

### ISSUE THREAD

**Issue by kumasansord** [bootscore/bootscore#71]
Off canvas on mobile menu closes by ios safari

**Comment by crftwrk** [comment]
Yes, Safari is always a struggle. This bug was shipped with iOS 15. 

It seems that body scroll is not properely locked if offcanvas is fullscreen. Offcanvas closes because body scroll is still active. To explain, please do this:

- Open site on iPhone Safari in portrait mode and open offcanvas menu. Most times it will close when you try to scroll offcanvas-body.
- Turn phone to landscape mode and open offcanvas menu again. Now body scroll is locked and you can scroll offcanvas body without any problems.

This bug is only on iPhone Safari in portrait mode, landscape is fine, iPad is all fine.

In conclusion, we address this to Apple or Bootstrap.
